projects/angular-calendar/src/modules/common/keydown-enter.directive.ts

```diff
@@ -1,17 +1,46 @@
-import { Directive, Output, EventEmitter, HostListener } from '@angular/core';
+import {
+  Directive,
+  Output,
+  EventEmitter,
+  ElementRef,
+  NgZone,
+  Renderer2,
+  OnDestroy,
+} from '@angular/core';
 
 @Directive({
   selector: '[mwlKeydownEnter]',
 })
-export class KeydownEnterDirective {
-  @Output('mwlKeydownEnter') keydown = new EventEmitter<any>(); // tslint:disable-line
+export class KeydownEnterDirective implements OnDestroy {
+  @Output('mwlKeydownEnter') keydown = new EventEmitter<KeyboardEvent>(); // tslint:disable-line
 
-  @HostListener('keydown', ['$event'])
-  onKeyPress(event: any) {
-    if (event.keyCode === 13 || event.which === 13 || event.key === 'Enter') {
-      event.preventDefault();
```

Comment:
I think we need to keep the preventDefault and stopPropagation?

Reply:
Done